cli/cli_app.go

```diff
@@ -61,6 +62,14 @@ func CreateTerragruntCli(version string) *cli.App {
    Moreover, for the apply and destroy commands, Terragrunt will first try to acquire a lock using DynamoDB. For
    documentation, see https://github.com/gruntwork-io/terragrunt/.`
 
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "config",
+			Value: os.Getenv("TERRAGRUNT_CONFIG"),
```

Comment:
Hi there, been following this thread for a while. My personal opinion, I think it'd be much better if instead of setting an ENV var, you could do something like

`terragrunt apply my/path/to/terragrunt/`

Which would be more or less the same as:
`terraform apply my/dir`